### PR TITLE
Ocean Hack Week Updates

### PR DESCRIPTION
**Pull request by ConnorFlynn**
🌎 Temp and Salinity Data from World Ocean Atlas for all Locations
- thinking about the role oceanexplorer package can play in this
🌎 Temp and Salinity Comparison for 3 locations
- North Pacific, South Pacific, Brazil
🌎 Next Steps:
- Standard Dev and Raw Difference to supplement % difference in model comparison
- Depth slider
- Data for other locations

### CODE REVIEW

**Comment by lidefi87** [comment]
Thanks so much for sending this through and for all the awesome work you both did last week 😁 

I will be making small changes to this PR so it works well with the app before merging it. I am just updating my repo copy right now

File: README.md
Comment (lidefi87) [inline README.md:1]: needs to be changed back to original text

**Review by lidefi87 — APPROVED** [review]

**Comment by lidefi87** [comment]
Thanks again @ConnorFlynn and @stormeyseas for your help with this. I have added you as co-authors of the app. If you like, you can add your ORCID to the `CITATION.cff` file via PR or you can share it with me and I will add it.

**Comment by stormeyseas** [comment]
Cool, thanks! My OrcID is 0000-0002-5983-2798. I'm actually keen to keep contributing to the FishMIP app if you're still working on it 😊

**Comment by lidefi87** [comment]
Hi @stormeyseas, great to hear you still want to be involved. I will have a chat to Julia to see how we can arrange this and I will get back to you.

I also added your ORCID to the citation file 😊

**Comment by stormeyseas** [comment]
Fantastic, thanks!

Regards,

Dr Tormey Reimer
She/her | 0419 679 495

*I acknowledge the Traditional Owners of the lands and waters on which I
work and live, and pay my respects to the Elders, past, present and
emerging. Sovereignty was never ceded.*

**Comment by ConnorFlynn** [comment]
Hey @lidefi87 thanks so much, the app is looking great! My OrcID is 0009-0004-8407-7124. I got to share some of the work you do with the undergraduates and they were very inspired! I've also finally filled out the event feedback form

**Comment by lidefi87** [comment]
I am glad your students liked it 😊 I am still trying to make it more efficient, but it seems progress is rather slow, but I'll get there.

Also, I just added your ORCID to the citation file.

**Pull request merged**